Update docs for API-only users (#45606)
fleetdm/fleet: articles/foreign-vitals-map-idp-users-to-hosts.md

3. For the **Unique identifier field for users**, enter `userName`.
4. For the **Supported provisioning actions**, select **Push New Users**, **Push Profile Updates**, and **Push Groups**.
5. For the **Authentication Mode**, select **HTTP Header**.

> 6. Create a Fleet API-only user with maintainer permissions (on Fleet Premium, head to **Settings > Users > Create user** and select the **API-only** option) and copy API token for that user. Paste your API token in Okta's **Authorization** field.

Reviewer: ```suggestion
6. [Create a Fleet API-only user](https://fleetdm.com/guides/fleetctl#create-api-only-user) with maintainer permissions and copy API token for that user. Paste your API token in Okta's **Authorization** field.
```

Free too right? So we don't need a change here?

5. For the **Authentication Mode**, select **HTTP Header**.
6. [Create a Fleet API-only user](https://fleetdm.com/guides/fleetctl#create-api-only-user) with maintainer permissions and copy API token for that user. Paste your API token in Okta's **Authorization** field.

> > You can also create the user from the command line: `fleetctl user create --name 'SCIM User' --api-only --global-role maintainer`

Author: Do we need note this now that we have UI to create API-only users?

Reviewer: Good call, I think we can pull it out. Just did.

> > You can also create the user from the command line: `fleetctl user create --name 'SCIM User' --api-only --global-role maintainer`

Reviewer: ```suggestion
```

1. From the side menu, select **Provisioning**.
2. In **Get started with application provisioning** section, select **Connect your application**.
3. For the **Tenant URL**, enter `https://<your_fleet_server_url>/api/v1/fleet/scim?aadOptscim062020`.

> 4. Create a Fleet API-only user with maintainer permissions (on Fleet Premium, head to **Settings > Users > Create user** and select the **API-only** option) and copy API token for that user. Paste your API token in the **Secret token** field.

Reviewer: ```suggestion
4. [Create a Fleet API-only user](https://fleetdm.com/guides/fleetctl#create-api-only-user) with maintainer permissions and copy API token for that user. Paste your API token in the **Secret token** field.
```

Same as here: https://github.com/fleetdm/fleet/pull/45606/changes#r3250392861

1. From the side menu, select **Applications > Providers**, **Create**, **SCIM Provider**, and then **Next**.
2. Add a friendly name (e.g. "Fleet SCIM provider").
3. For the **URL**, enter `https://<your_fleet_server_url>/api/v1/fleet/scim`.

> 4. Create a Fleet API-only user with maintainer permissions (on Fleet Premium, head to **Settings > Users > Create user** and select the **API-only** option) and copy the API token for that user. Paste your API token in the **Token** field.

Reviewer: ```suggestion
4. [Create a Fleet API-only user](https://fleetdm.com/guides/fleetctl#create-api-only-user) with maintainer permissions and copy API token for that user. Paste your API token in the **Secret token** field.
```

IdPs generally require a Fleet SCIM URL and API token:

- SCIM URL - `https://<your_fleet_server_url>/api/v1/fleet/scim`

> - API token - Create a Fleet API-only user with maintainer permissions (on Fleet Premium, head to **Settings > Users > Create user** and select the **API-only** option) and copy API token for that user.

Reviewer: ```suggestion
- API token - [Create a Fleet API-only user](https://fleetdm.com/guides/fleetctl#create-api-only-user) with maintainer permissions and copy API token for that user. Paste your API token in the **Secret token** field.
```
